dev/requirements.txt

```diff
@@ -1,7 +1,6 @@
-python=3.6
 pycosat >=0.6.3
 pyopenssl >=16.2.0
 requests >=2.12.4,<3
 ruamel_yaml >=0.11.14
 git
-
+conda-package-handling
```

Comment:
I think this requirement also needs to be added to `utils/requirements.txt` so that docs stuff (read the docs and the gh-pages tool) don't break

Reply:
we currently can't.  As we discussed in person, conda-package-handling requires libarchive, and I haven't been able to get that working with the pypi-provided libarchive.  I think there's also a problem with our python-libarchive-c package such that pip doesn't see that it is providing the libarchive package.

Overall, yes, I agree with you, but unfortunately, I'm not sure we can. =(